superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterControl.tsx

```diff
@@ -142,7 +158,24 @@ const HorizontalFormItem = styled(StyledFormItem)`
   }
 
   .ant-form-item-control {
-    width: ${({ theme }) => theme.gridUnit * 41}px;
+    width: ${({ theme, inverseSelection }) =>
+      inverseSelection ? theme.gridUnit * 41 * 1.6 : theme.gridUnit * 41}px;
+  }
+
+  .ant-form-item-control-input-content {
+    ${({ inverseSelection }) => inverseSelection && 'display: flex;'}
+  }
+
+  .select-container {
+    ${({ inverseSelection, theme }) =>
+      inverseSelection &&
+      `
+      width: ${theme.gridUnit * 41}px;
```

Comment:
No `gridUnit` for widths, please. I know this was like this before but it is likely a mistake.